### ISSUE TITLE
Option to select only specific targets for installation

### ISSUE THREAD

**Issue by georgik** [espressif/idf-installer#23]
Add an option to select targets for the installation so that the installation size is smaller and toolchains which are not required for the target/series is not installed.

**Comment by georgik** [comment]
idf_tools throws error in case of unknown parameter:
```
Running command: C:\projects\tmp\.espressif\tools\idf-python\3.8.7\python.exe "C:\Users\USER\Desktop\esp-idf-4.1.1\tools\idf_tools.py" --idf-path C:\Users\USER\Desktop\esp-idf-4.1.1\ --targets=esp32,esp32-c3,esp32-s3,esp32-s2 install
                          usage: idf_tools.py [-h] [--quiet] [--non-interactive]
                                              [--tools-json TOOLS_JSON] [--idf-path IDF_PATH]
                                              {list,check,export,install,download,install-python-env}
                                              ...
                          idf_tools.py: error: unrecognized arguments: --targets=esp32,esp32-c3,esp32-s3,esp32-s2
```
There must be also additional check in installer whether idf_tools has the support for selecting target.

**Comment by georgik** [comment]
The list of enabled components should be filtered based on 
https://dl.espressif.com/dl/esp-idf/idf_versions.js

**Comment by georgik** [comment]
The feature is not ready in ESP-IDF side, postponing GUI part for next release.
FYI: @mfialaf

**Comment by georgik** [comment]
Solved by #24
